taskcluster_util/taskcluster_download.py

```diff
@@ -102,6 +102,15 @@ def _check_crendentials_file(self, options):
             logger.info('Load Credentials from {}'.format(abs_credentials_path))
         except Exception as e:
             logger.warning('No credentials. Run with "--help" for more information.')
```

Comment:
This message is always printed out, even though you gave a credential file. This looks odd. I'd move it in an `else`